src/Illuminate/Foundation/Testing/DatabaseMigrations.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Illuminate\Foundation\Testing;
 
+use Illuminate\Foundation\Console\Kernel;
```

Comment:
We might want to use Illuminate\Contracts\Console\Kernel here... I think that is what is bound to the container when the app is bootstrapped.